source/Controls.hx

```diff
@@ -563,6 +567,30 @@ class Controls extends FlxActionSet
 		#end
 	}
 
+	static function loadControls(){
+		if (FlxG.save.data.keyboardMap != null)
+		{
+			keyboardMap = FlxG.save.data.keyboardMap;
+		}
```

Comment:
Won't doing it like this mean that `controls.txt` isn't used after the first time you launch the game? At that point you could probably just delete it entirely and have the first run generate default controls in this file.

Reply:
Well yes, and honestly I just took the lazy way out. It could be a better idea to remove controls.txt, but as long as it works I don't believe it's too necessary.

Reply:
Leaving controls.txt in is probably misleading to the user since it gives the impression that it can be edited to change controls outside of the game

Reply:
Finally getting back to this, I think that renaming the file to "default controls" would help prevent such a misinterpretation.